service/alerter/metric.go

```diff
@@ -3,6 +3,15 @@ package alerter
 import "github.com/prometheus/client_golang/prometheus"
 
 var (
+	duplicateResourcesTotal = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "awsoperator",
+			Subsystem: "resources",
+			Name:      "duplicate_resources_total",
+			Help:      "Number of clusters with duplicate resources to be cleaned up.",
+		},
+		[]string{"resource"},
+	)
```

Comment:
Joe, this is the new metric. Does it look OK? Also are the more steps I need to get it scraped by Prometheus?

Reply:
You need to register the metric with prometheus, see https://github.com/giantswarm/aws-operator/blob/f683ebbc4fd7370640242f47a9e9d3c71b7467b7/service/alerter/metric.go#L27

Reply:
I don't see the register?

Reply:
Thanks, its there now.